### PR TITLE
spring-boot-2-classifier addition

### PR DESCRIPTION
**Pull request by mohshabbir**
added classifier

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -72,6 +72,24 @@
 
 	<build>
 		<plugins>
+
```
Comment (MALPI) [inline pom.xml]: Please remove that blank line.

**Comment by MALPI** [comment]
👍

**Comment by MALPI** [comment]
@mmodabber @coders-kitchen please review/approve

**Comment by coders-kitchen** [comment]
👍

**Comment by MALPI** [comment]
Sorry but the effort resolving the conflicts are too high. Therefore I made the addition on the branch once more. Thanks anyways for your contribution!

**Pull request closed without merging**